alifedata_phyloinformatics_convert/ALifeTree.py

```diff
@@ -0,0 +1,61 @@
+from .alife_dataframe_to_biopython_tree \
+    import alife_dataframe_to_biopython_tree
+from .alife_dataframe_to_dendropy_tree \
+    import alife_dataframe_to_dendropy_tree
+from .biopython_tree_to_alife_dataframe \
+    import biopython_tree_to_alife_dataframe
+from .dendropy_tree_to_alife_dataframe \
+    import dendropy_tree_to_alife_dataframe
+
+import pandas
+import dendropy
+import Bio
+
+# auxiliary tree class to allow for inter-format conversions
+class ALifeTree():
```

Comment:
renaming ideas: PolyTree, FlexTree, AdapterTree, SwisArmyTree, MultiTree (not AnyTree, that's already a tree package), PolyglotTree, BabelTree, OmniTree

Reply:
I think we should rename to emphasize that it should be used for getting views in different formats, not the association with the alife format

Reply:
VersiTree ViewableTree CastingTree RecastingTree

Reply:
AllTree

Reply:
LexicalTree

Reply:
MotleyTree PanTree

Reply:
I kind of like PanTree and OmniTree, pantree makes me think of pandoc

Reply:
Another cute name could be RosettaTree I like that one too (maybe the most)

Reply:
Renamed in d4d3e94